nginx/nginx.conf

```diff
@@ -43,11 +43,24 @@ http {
 	proxy_next_upstream error timeout invalid_header http_502 http_503 non_idempotent;
 	proxy_next_upstream_tries 2;
 
+	# Docker default address pools
+	# https://github.com/moby/libnetwork/blob/3797618f9a38372e8107d8c06f6ae199e1133ae8/ipamutils/utils.go#L10-L22
+	set_real_ip_from 172.17.0.0/16;
+	set_real_ip_from 172.18.0.0/16;
+	set_real_ip_from 172.19.0.0/16;
+	set_real_ip_from 172.20.0.0/14;
+	set_real_ip_from 172.24.0.0/14;
+	set_real_ip_from 172.28.0.0/14;
+	set_real_ip_from 192.168.0.0/16;
+	set_real_ip_from 10.0.0.0/8;
+	real_ip_header X-Forwarded-For;
+	real_ip_recursive on;
+
 	# Remove the Connection header if the client sends it,
 	# it could be "close" to close a keepalive connection
 	proxy_set_header Connection '';
 	proxy_set_header Host $host;
-	proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+	proxy_set_header X-Forwarded-For $remote_addr;
```

Comment:
Note: this might break some installations that have multiple hops before this Nginx, e.g. when the self-hosted installation is behind Google/AWS load balancers that normally add a bunch of records to `X-Forwarded-For` header automatically. 

That means that we should highlight this in the release notes, and point to recommendations around "how to pass the trusted client IP to the Sentry backend".